contracts/VetoERC20Voting.sol

```diff
@@ -132,7 +131,6 @@ contract VetoERC20Voting is
 
     /// @notice Unfreezes the DAO, only callable by the owner
     function defrost() public onlyOwner {
-        require(isFrozen(), "DAO is not already frozen");
```

Comment:
this makes sense you don't actually need this. If it's not frozen it's just setting 0s to 0s in any case right?

Reply:
Well they may not necessarily be 0's. It could be non-zero, but just not above the required threshold to freeze. So removing this did change the functionality a bit, but I think it makes sense to give the owner (parent DAO) this ability.

Reply:
yeah fair enough, the parent DAO can do pretty much anything I guess.  Would that mean they can basically cut short a freeze vote mid vote if they'd like to?

Reply:
Yep, that's correct